oauth2client/client.py

```diff
@@ -1617,6 +1617,23 @@ def _revoke(self, http_request):
         """
         self._do_revoke(http_request, self.access_token)
 
+    def sign_blob(self, blob):
+        """Cryptographically sign a blob (of bytes).
+
+        Args:
+            blob: bytes, Message to be signed.
+
+        Returns:
+            tuple, A pair of the private key ID used to sign the blob and
+            the signed contents.
+        """
+        raise NotImplementedError
+
+    @property
```

Comment:
Perhaps a moot point by now but for my own curiosity: why `@property` and not `@abstractproperty`?

Fun fact about me: I don't like, and dispute the premise of, both `abstractproperty` and `property`.